code/game/objects/items/syndicateReverseCard.dm

```diff
@@ -0,0 +1,54 @@
+
+//Syndie Uno reverse card
+/obj/item/syndicateReverseCard
+	name = "Red Reverse"
+	icon = 'icons/obj/toy.dmi'
+	icon_state = "sc_Red Reverse_uno"
+	desc = "a card."
+	w_class = WEIGHT_CLASS_TINY
+	var/used = FALSE //has this been used before? If not, give no hints about it's nature
+
+/obj/item/syndicateReverseCard/Initialize()
+	..()
+	var/cardColor = pick ("Red", "Green", "Yellow", "Blue") //this randomizes which color reverse you get!
+	name = "[cardColor] Reverse"
+	icon_state = "sc_[cardColor] Reverse_uno"
+
+/obj/item/syndicateReverseCard/update_icon()
+	..()
+	if (used)
+		cut_overlays()
+		add_overlay(image('icons/obj/items_and_weapons.dmi', icon_state = "Reverse_Overlay"))
```

Comment:
icon states should be in all lowercase whenever possible.

Also you have a white pixel in the top left corner in the last 5 frames

Reply:
Ignore the second part